config/conf.go

```diff
@@ -12,12 +12,25 @@ import (
 // This is only needed because if "instance". Otherwise we would just run
 // RegisterAlias inside the BindPFlags function.
 var configToFlagMap = map[string]string{}
+var confDirs = []string{".", DefaultConfDir}
+var confName = DefaultConfName
+
+// SetConfDirs sets the directories to search for the config file when a file
+// name is not explicitly provided. If a file name is explicitly provided viper
+// checks both relative and absolute file paths.
+func SetConfDirs(dirs []string) {
+	confDirs = dirs
+}
+
+// SetConfName sets the name of the config file to search for. For aerospike
+// tools this is astools but for asvec it is asvec.
+func SetConfName(name string) {
```

Comment:
I think this should be called `SetDefaultConfName` since it is ignored if the user provides a config file name.